src/Raven.Server/Smuggler/DatabaseDataExporter.cs

```diff
@@ -30,16 +33,20 @@ public ExportResult Export(DocumentsOperationContext context, string destination
 
         public ExportResult Export(DocumentsOperationContext context, Stream destinationStream)
         {
-            long lastDocsEtag = 0;
+            var result = new ExportResult();
 
             using (var gZipStream = new GZipStream(destinationStream, CompressionMode.Compress, leaveOpen: true))
             using (var writer = new BlittableJsonTextWriter(context, gZipStream))
             {
                 writer.WriteStartObject();
 
+                writer.WritePropertyName(context.GetLazyString("BuildVersion"));
+                writer.WriteInteger(40000);
```

Comment:
If we don't support exporting from v4.0 and improting to v3.5 that it doesn't matter. So for now I leave it as a number, and we'll check later on how this is working with importing to v3.5.